### ISSUE TITLE
[Feature request]Support for APNG to Animated WebP?

### ISSUE THREAD

**Issue by dreampiggy** [iSparta/iSparta#13]
[Animated WebP](https://developers.google.com/speed/webp/docs/riff_container) is an modern and open source image format, it use VP8 encoder and reduce the size fo image. WebP has animated image support, and also a full alpha channel compared to GIF.

For convertor from APNG to Animated WebP, here an many open source project, and an command line tool: https://github.com/Benny-/apng2webp. Which can cross-platform build using Python and some C++ library.

Our application use this tool to convert the APNG image generated from iSparta to animated WebP. Finally it reduce nearly 20% image size and get cross-platform support. Becasue actually APNG for Android can't support well, But for Animated WebP, you can use [YYImage](https://github.com/ibireme/YYImage) for iOS and [Fresco](https://github.com/facebook/fresco) for Android(Also with Chromium WebView build-in support). 

But at now, we just use iSparta GUI to create APNG and run a shell script to build Animated WebP, which seems strange for UI and UE to do this.

So, it's any plan to support APNG to Animated WebP automatically?

**Comment by dreampiggy** [comment]
Pull Request is here: #14 

I'm not sure whether this project is alive or not (@jeakey @hahnzhu) , but I hope that there are someone who can merge this or create an release build for those who want apng to animated webp convertor. 😄 

What I did:

+ Add apng2webp convertor, thantks to https://github.com/Benny-/apng2webp. These binary is published with zlib license, if you'd like, I will add license on README.md
+ Major changes are the apng2webp related js, html and css
+ Also, dure to the fact that the convertor just use `cwebp` and `webpmux` but not directly link to `libwebp`, I replace them with the newest libwebp 6.0's binary.
+ Finally I fix some experience issues, such as imagelossless only check png files(but actually it can compress png, jpeg and gif images).

**Comment by dreampiggy** [comment]
For localization(i18n) support, I tried some ways but with no luck. Because of this old version `node-webkit` framework(3 years ago?) can not works if you run it as a node-server(`node-main` is entry for `app.js` running express, and `main` is `http://localhost:3000`). For i18n support, we should either rewrite this project with an full-stack JavaScript framework(Angualr Vue) instead of simple HTML based render or we should choose server side rendering.

After I tried the latest `nw.js` release, it can work, and I use [i18n-node](https://github.com/mashpie/i18n-node) with doT template to render that `main.html` template for multiple language. But there are somethinig strange issue(for example, the /lib folder sometimes not response). So I finally gave up.

My translation is here: 
[zh-CN.json.zip](https://github.com/iSparta/iSparta/files/1152244/zh-CN.json.zip)
I hope this is useful if someone else want to add i18n support to iSparta. 😀
